src/main/resources/interface-specification.yml

```diff
@@ -214,6 +214,54 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  '/clients/{clientId}':
+    get:
+      tags:
+        - gateway
+      summary: Get a client
+      description: Retrieve a client using a client identifier
+      operationId: getClient
+      security:
+        - bearerAuth: [ ]
+      parameters:
+        - name: clientId
+          in: path
+          description: the client Id
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: Client retrieved
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Client'
+        '400':
+          description: Bad request
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+        '401':
+          description: Unauthorized
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+        '403':
```

Comment:
about the suggestion to return a 404 in case of unauthorised request, which approach are we using?

Reply:
The discussion about this seems stuck.
I suggest keeping 403 hiding information about what happened.
Eventually, this error will be changed if necessary.